packages/config/config/devices/0x0371/zwa024.json

```diff
@@ -0,0 +1,832 @@
+				},
+				{
+					"label": "Set 0x00 on triggered, 0xff on cleared",
+					"value": 1
+				},
+				{
+					"label": "Set 0xff on motion triggered",
+					"value": 2
+				},
+				{
+					"label": "Set 0x00 on motion triggered",
+					"value": 3
+				},
+				{
+					"label": "Set 0x00 on motion cleared",
+					"value": 4
+				},
+				{
+					"label": "Set 0xff on motion cleared",
+					"value": 5
+				},
+				{
+					"label": "As configured in parameter 12",
+					"value": 6
+				}
+			]
+		},
+		{
+			"#": "12",
+			"label": "Direct Association (Motion): Basic Set Value (Custom)",
+			"description": "Define a custom value to be sent when motion is detected; parameter 11 must be set to follow parameter 12",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 65535,
+			"defaultValue": 65280,
+			"unsigned": true
+		},
+		{
+			"#": "13",
+			"label": "Direct Association (Motion): Luminance Threshold",
+			"valueSize": 2,
+			"unit": "lux",
+			"minValue": 0,
+			"maxValue": 30000,
+			"defaultValue": 30000,
+			"unsigned": true
+		},
+		{
+			"#": "14",
+			"label": "Direct Association (High Humidity): Basic Set Threshold",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 60,
+			"unsigned": true
+		},
+		{
+			"#": "15",
+			"label": "Direct Association (Low Humidity): Basic Set Threshold",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 40,
+			"unsigned": true
+		},
+		{
+			"#": "16",
+			"label": "Direct Association (High Temperature): Basic Set Threshold",
+			"valueSize": 2,
+			"unit": "0.1 °C/°F",
+			"minValue": -400,
+			"maxValue": 1850,
+			"defaultValue": 750
+		},
+		{
+			"#": "17",
+			"label": "Direct Association (Low Temperature): Basic Set Threshold",
+			"valueSize": 2,
+			"unit": "0.1 °C/°F",
+			"minValue": -400,
+			"maxValue": 1850,
+			"defaultValue": 600
+		},
+		{
+			"#": "18",
+			"label": "Direct Association (High Luminance): Basic Set Threshold",
+			"valueSize": 2,
+			"unit": "lux",
+			"minValue": 0,
+			"maxValue": 30000,
+			"defaultValue": 2000,
+			"unsigned": true
+		},
+		{
+			"#": "19",
+			"label": "Direct Association (Low Luminance): Basic Set Threshold",
+			"valueSize": 2,
+			"unit": "lux",
+			"minValue": 0,
+			"maxValue": 30000,
+			"defaultValue": 100,
+			"unsigned": true
+		},
+
+		{
+			"#": "20",
+			"label": "Direct Association (High UV): Basic Set Threshold",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 11,
+			"defaultValue": 8,
+			"unsigned": true
+		},
+		{
+			"#": "21",
+			"label": "Direct Association (Low UV): Basic Set Threshold",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 11,
+			"defaultValue": 1,
+			"unsigned": true
+		},
+		{
+			"#": "40",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Change Threshold Reporting Reporting"
+		},
+		{
+			"#": "45",
+			"label": "Change Threshold Reporting: Check Interval",
```

Comment:
I think we've called `Change Threshold Reporting` `Reporting Threshold` elsewhere. IMO this is one of the things we should normalize as a next step.

Reply:
There are two different things here, though. I've been using Reporting Threshold where it reports when your cross the set threshold. The Change Threshold is where it reports when there is a particular degree of change from the last.